config/owasp/suppressions.xml

```diff
@@ -15,7 +15,7 @@
     <cve>CVE-2018-10237</cve>
   </suppress>
 
-  <suppress until="2021-03-01">
+  <suppress until="2021-05-01">
```

Comment:
you can remove this entry it is same with newly added one

Reply:
Remved.